packages/documentation/pages/usage/components/card.vue

```diff
@@ -1,7 +1,49 @@
 <template>
 	<div>
 		<ComponentInfo v-bind="{ component }" />
+		<div class="catalog-tile-view-example">
```

Comment:
Please rename this to something more generic like:

```suggestion
		<div class="card-view-example">
```